.laminas-ci.json

```diff
@@ -5,6 +5,7 @@
         "intl"
     ],
     "ignore_php_platform_requirements": {
+        "8.0": true,
         "8.1": true
```

Comment:
I agree with that, but we'd have to get 8.0 and 8.1 supported in laminas-cli to do that wouldn't we?

Reply:
Is laminas-cli even involved here?

I think `laminas-console` is still in here - we could update that to be compliant, since it's probably less work than releasing a new major version here :thinking:

Reply:
Sorry, yes, I meant `laminas-console`

Reply:
Yes, let's leave it be then :+1: